### DISCUSSION TITLE
Bitcoin Code

### DISCUSSION THREAD

**Discussion by yasharyan0904** [Cyfrin/foundry-full-course-cu#1900] [Q&A]
Can anyone suggest me any resource from where I can understand the fundamentals of the Bitcoin Code, its working and everything?

**Comment by EngrPips** [comment]
Seem like you need to read the bitcoin white-paper `https://bitcoin.org/bitcoin.pdf`

**Reply by yasharyan0904** [reply]
Read :) for Ethereum too.
Just wanted to go through the code and a bit of explanation, if any resource you could suggest?

**Reply by yasharyan0904** [reply]
I am currently reading "The Basis of BITCOINS and Blockchains" by Antony Lewis. Anyone interested in understanding the theoretical aspect can read it.

**Reply by EngrPips** [reply]
Interesting. There is so much stuff to read but so little time at hand. Argh

**Reply by EngrPips** [reply]
> Read :) for Ethereum too. Just wanted to go through the code and a bit of explanation, if any resource you could suggest?

I don't know of any right now but if I see any I will surely suggest it.

**Reply by yasharyan0904** [reply]
Great :)
